modules/bitgo/test/v2/unit/lightning.ts

```diff
@@ -84,4 +84,33 @@ describe('lightning API requests', function () {
     assert.deepStrictEqual(res, fixtures.invoices);
     scope.done();
   });
+  
+  it('should decode lnurl-pay', async function () {
+    const lnurl = 'LNURL1DP68GURN8GHJ7UM9WFMXJCM99E3K7MF0V9CXJ0M385EKVCENXC6R2C35XVUKXEFCV5MKVV34X5EKZD3EV56NYD3HXQURZEPEXEJXXEPNXSCRVWFNV9NXZCN9XQ6XYEFHVGCXXCMYXYMNSERXFQ5FNS';
```

Comment:
Please add the code that generates the LNURL somewhere and use the generated result here instead. This makes it easier to verify what's going on.